app/uploaders/profile_photo_uploader.rb

```diff
@@ -1,17 +1,23 @@
 # encoding: utf-8
 
 class ProfilePhotoUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
   if Rails.env.test?
     storage :file
   else
     storage :fog
   end
 
+  version :thumb do
+    process resize_to_fill: [100, 100]
+  end
+
   def store_dir
     "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
-  def default_url
+  def default_url(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `default_url(*)` if you want the method to accept any arguments but don't care about them.